packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -460,8 +460,13 @@ const getAnswersFromStdin = async () => {
 /**
  * Prints the contents of program.json in a nicely formatted way and exits the process.
  */
-const printProgramJsonAndExit = () => {
-    console.log(JSON.stringify(PROGRAM, null, 2))
+const printProgramJsonAndExit = async () => {
+    const output = JSON.stringify(PROGRAM, null, 2)
+    await new Promise((resolve) => {
+        process.stdout.write(output + '\n', () => {
+            resolve()
+        })
+    })
```

Comment:
This was a bug that I originally made where the output was being printed and the program exited before the standard out could be flushed. This fixes that problem.